instrumentation/go.mongodb.org/mongo-driver/v2/mongo/otelmongo/mongo.go

```diff
@@ -64,12 +76,57 @@ func (m *monitor) Started(ctx context.Context, evt *event.CommandStartedEvent) {
 	m.Unlock()
 }
 
-func (m *monitor) Succeeded(_ context.Context, evt *event.CommandSucceededEvent) {
+func (m *monitor) Succeeded(ctx context.Context, evt *event.CommandSucceededEvent) {
 	m.Finished(&evt.CommandFinishedEvent, nil)
+
+	hostname, port := peerInfo(evt.ConnectionID)
+	attrs := []attribute.KeyValue{
+		// No need to add semconv.DBSystemMongoDB, it will be added by metrics recorder.
+		semconv.DBOperationName(evt.CommandName),
+		semconv.DBNamespace(evt.DatabaseName),
+		semconv.NetworkPeerAddress(hostname),
+		semconv.NetworkPeerPort(port),
+		semconv.NetworkTransportTCP,
+		semconv.DBResponseStatusCode(statusCodeOK),
```

Comment:
MongoDB doesn't have a "status code" in the HTTP tradition. Instead, it returns `{ok: 1}`. For example:

```
{"n": {"$numberInt":"1"},"electionId": {"$oid":"7fffffff0000000000000001"},"opTime": {"ts": {"$timestamp":{"t":1762377533,"i":1}},"t": {"$numberLong":"1"}},"ok": {"$numberDouble":"1.0"},"$clusterTime": {"clusterTime": {"$timestamp":{"t":1762377533,"i":1}},"signature": {"hash": {"$binary":{"base64":"AAAAAAAAAAAAAAAAAAAAAAAAAAA=","subType":"00"}},"keyId": {"$numberLong":"0"}}},"operationTime": {"$timestamp":{"t":1762377533,"i":1}}}
```

 I'm hesitant simplifying this to "OK". Suggest not reporting this value on successful commands, it seems unnecessary.